lib/ansible/modules/hostname.py

```diff
@@ -36,6 +36,7 @@
         description:
             - Which strategy to use to update the hostname.
             - If not set we try to autodetect, but this can be problematic, particularly with containers as they can present misleading information.
+            - Note that 'systemd' should be specified for RHEL/EL 7+. Older distributions should use 'redhat'.
```

Comment:
```suggestion
            - For RHEL/EL 7+, specify 'systemd'. For older RHEL/EL versions, specify 'redhat'.
```

Reply:
not just RHEL, but 'those derived from it' ... for example CentOS

Reply:
Are there any other major distributions that are downstream from RHEL besides CentOS that you can think of where this would apply?

Reply:
Settled on RHEL/EL/CentOS.